contributors/devel/sig-instrumentation/migration-to-structured-logging.md

```diff
@@ -467,16 +477,17 @@ func ChangePodStatus(newStatus, currentStatus string) {
 ## Good practice for passing values in structured logging
 
 When passing a value for a key-value pair, please use following rules:
-* Prefer using Kubernetes objects (for example `*v1.Pod`) and log them using `klog.KObj`
+* Prefer using Kubernetes objects (for example `*v1.Pod`) and log them using `klog.KObj` or `klog.KObjSlice`
   * When the original object is not available, use `klog.KRef` instead
+  * When object sets, we use `klog.KObjSlice`
```

Comment:
I think this line needs a bit more clarification. Did you mean `When the object is slice type` ?